test/ui/roles/detail/DetailRolePage.test.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import 'test/enzyme-init';
+import { shallow } from 'enzyme';
+import DetailRolePage from 'ui/roles/detail/DetailRolePage';
+
+describe('DetailRolePage', () => {
+  let component;
+  beforeEach(() => {
+    component = shallow(<DetailRolePage />);
+  });
+  it('renders without crashing', () => {
+    expect(component.exists()).toEqual(true);
+  });
+
+  it('verify if exist InternalPage with class DetailRolePage', () => {
+    expect(component.find('InternalPage').hasClass('DetailRolePage')).toEqual(true);
+  });
+
+  it('verify if has a breadcrumb', () => {
+    expect(component.find('Breadcrumb').exists());
```

Comment:
This isn't actually testing anything. The correct test would be

`expect(component.find('Breadcrumb')).toExist();`

or

`expect(component.find('Breadcrumb').exists()).toBe(true);`